graphene_sqlalchemy/tests/test_types.py

```diff
@@ -181,3 +181,36 @@ class Meta:
         resolver, TestConnection, ReporterWithCustomOptions, None, None
     )
     assert result is not None
+
+
+class HumanWithFieldAliased(SQLAlchemyObjectType):
+
+    publication_timestamp = Int()
+
+    class Meta:
+        model = Article
+        registry = registry
+        interfaces = (Node,)
+        aliased_fields = {
+            "id": "article_id",
+            "headline": "title",
+            "pub_date": "publication_timestamp",
+            "reporter_id": "journalist_id",
+            "reporter": "journalist",
+        }
+
+
+def test_objecttype_with_aliased_fields():
+    assert issubclass(HumanWithFieldAliased, ObjectType)
+    assert HumanWithFieldAliased._meta.model == Article
+    assert list(HumanWithFieldAliased._meta.fields.keys()) == [
```

Comment:
You don't need the `.keys()` here because a dict is iterable itself. Normally you would need to sort the keys to get a robust test, but in this case we have an ordered dict so this is ok.